docs/versus.md

```diff
@@ -259,7 +259,7 @@ symbol legend,
   * you can successfully play `$'\355\221'` with mpv through mounting a remote copyparty server with rclone, pog
 * `a`/copyparty remarks:
   * extremely minimal samba/cifs server
-  * netscape 4 / ie6 support is mostly listed as a joke altho some people have actually found it useful ([ie4 tho](https://user-images.githubusercontent.com/241032/118192791-fb31fe00-b446-11eb-9647-898ea8efc1f7.png))
```

Comment:
altho -> although but not tho -> though?

Reply:
;-) nice observation.  up to you -- we could exclude `altho` for symmetry or I could suggest `tho -> though` to codespell and fix here. Your desire?